### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wolfstudy**

### CODE REVIEW

**Comment by wolfstudy** [comment]
the scripts directory exists in `charts` instead of `charts/charts`

File: README.md [hunk 1]
```diff
@@ -119,9 +119,6 @@ We provide some instructions to guide you through the preparation for the follow
     ```bash
     git clone https://github.com/streamnative/charts.git
     ```
-    ```bash
```
Comment (sijie) [inline README.md:122]: after cloning the repo, it will create a directory `charts`. So you need to enter the chart dir first, no?

Comment (wolfstudy) [reply README.md:122]: No, because the `scripts` directory does not exist in `charts/charts`.

**Comment by wolfstudy** [comment]
yes, this right

**Pull request closed without merging**